### DISCUSSION TITLE
Consent Management

### DISCUSSION THREAD

**Discussion by FabrizioMoggio** [camaraproject/CallForwardingSignal#7] [General]
is it required the consent from the end user for the API Consumer to invoke the Call FWD Signal API?

**Comment by FabrizioMoggio** [comment]
I think so.

**Comment by enlaichu** [comment]
Would the consent be verbally or through some user interaction such as clicking a link, reading a code sent via SMS/USSD or other mechanism?

**Reply by FabrizioMoggio** [reply]
the mechanism for acquiring the consent is done outside the API and before its invocation. You can refer to the document linked by @bigludo7 in the comment below. We should anyway have some deepening on this topic in some of our future calls. So that everyone is aligned  with how CAMARA is handling this topic.

**Comment by bigludo7** [comment]
Hello @FabrizioMoggio , @enlaichu , team
Got my meeting with my legal expert to ask for guidance.

From our side in Orange, we consider that the information about call forwarding must be considered as a personal data  (as linked to the line) and should be protected as such.
Depending on the purpose associated to the request, the legal base has to be determined. It is fair to assume that for a 'fraud detection & prevention' purpose we could use 'legitimate interest' legal base (other legal base could apply for other purpose...if any)

As such our recommendation is to follow the CAMARA 3-legs as described there: https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md#user-authenticationauthorization--consent-management

**Reply by FabrizioMoggio** [reply]
I agree.

**Reply by chinaunicomyangfan** [reply]
Whether to enable call forwarding should belong to personal sensitive information. I support using 3-leg authentication method

**Comment by FabrizioMoggio** [comment]
Supported by the current release
